domains/iystreem.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "IYSTREEM",
+           "email": "",
+           "discord": "774642006173286432",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.C64mBBovXkJe1xpUzL5ffA6LW_KuULNyvCgK85ZTegjPRDVAJGpQLRI6F9qUjG9K9CR2yrx15Po8P0LOxUdnKgTtqvSKRCutQJhxa64-wG35OZhJY26tW4Z5PJdUuVelZ2XhzJzO5pdqolxRxD065eLKriXCy2TfKrQV800C0fZcb7tiRC4XxzjJc3OOhE_uWmI9amMtgUtye8XW9LN1zffaiSESkcOjJjan6588i3iQyLS2fKoJPtzkX7fqDsW4soTDMRD5biqaDnmD0dKydJqM6Lay5JiZewo0k4NkP_ph6LCsUVjORp0EWctFXzG6-TIUU1WcQcLGViwAbicczQ.4e_zxFR5hXERQQpLuvs3Cw.qO7ynRvlUG_oPNpOU7OaT4cfciBaTCi2_Q65Eyu9WP3R7P7cPQBJRcU4QCKxdEx79nkus9W4jM8utnTW6WzG3p_K_0SwYuNYBVGz5GXf73k.f1Igi-sO1MQu7I15NVnspA"
+        },
+    
+        "record": {
+            "CNAME": "https://iystreem.freewebhostmost.com/"
```

CNAME can not contain `https://` or `/`
```suggestion
            "CNAME": "iystreem.freewebhostmost.com"
```